Yoast/wordpress-seo
# Deprecation of the Google Search Console code

## Summary


This PR can be summarized in the following changelog entry:

* Removal of Google Search Console related code.

## Relevant technical choices:

* I've put the deprecated version to 12.5, hopefully this is the right version. 

## Test instructions

This PR can be tested by following these steps:

* Make sure the Google Search Console and the Onboarding Wizard still works without showing any errors.

## UI changes

## Documentation

## Quality assurance

* [ ] I have ~added~ removed unittests to verify the code works as intended

Fixes #13289

You must merge https://github.com/Yoast/wordpress-seo-premium/pull/2577, otherwise Premium is kaputt.
